### PR TITLE
final tools to methods

### PR DESCRIPTION
**Pull request by mridulnagpal**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue
* [x] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer
* [ ] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.
#857 
Thanks!

### CODE REVIEW

**Comment by mridulnagpal** [comment]
screenshot from 2016-12-10 15 58 20
Title Changed and url as well

**Comment by jywarren** [comment]
Ooh, this is looking good -- reviewing now, thanks! Thanks for the screenshot, it's very helpful!

File: test/functional/notes_controller_test.rb [hunk 1]
```diff
@@ -95,8 +96,14 @@ def teardown
   test "should get tools" do
     get :tools
 
-    assert_response :success
-    assert_not_nil :notes
+    assert_response :redirect
```
Comment (jywarren) [inline test/functional/notes_controller_test.rb]: This all looks super, really great. I'd like to request an extra assertion here, though -- a `asserts_redirected_to '/methods'` so we're sure where the redirect goes to. Thanks!

**Comment by mridulnagpal** [comment]
@jywarren Yeah that test was required added that as well 
Cheers

**Comment by jywarren** [comment]
Super -- we'll be merging this after resolving some stufff in #1089 -- thanks a million! I'm curious if you could help 500swapnil out on #1083 if he has more questions there! 

In any case very happy to have your contribution and look forward to finishing up some of the other ones you've claimed. Thanks!

**Comment by jywarren** [comment]
Merged! 🎉

**Pull request merged**